thirdparty.mk

```diff
@@ -27,7 +28,8 @@ capnproto:
 	cmake . -B$(build) \
 	-DBUILD_SHARED_LIBS:BOOL=OFF \
 	-DBUILD_TESTING:BOOL=OFF \
-	-DCMAKE_BUILD_TYPE:STRING="Release" \
+	-DCMAKE_POSITION_INDEPENDENT_CODE=ON \
```

Comment:
position independent code is for shared libraries.
this will break existing workflow that builds librime and its dependencies as static libraries (mainly for Windows, macOS).